### PR TITLE
[Hack Update] 019-ThisOldDataWarehouse - Modernize ThisOldDataWarehouse What the Hack

### PR DESCRIPTION
**Pull request by DataSciNAll**
Here are the updates to this release.
1. Added two new challenges for Enterprise Security and Purview
2. Refactored hack to follow templates for What the Hack
3. Updated all links to point to new Synapse Analytics documentation
4.  Updated terminology to replace SQL DW and ADF with Synapse Analytics and pipelines.
5. Updated Success criteria for challenges to be less an instruction manual
6. Cleaned up coaches solutions and updated them to current capabilities

These updates were made from 07/22 to 11/22.   Our roadmap for the future will be;
1. Adding new challenges for handling parquet files
2. Refactor ADF jobs into Azure Data flows
3. Setting up restore files for CH1, 2, 3.  This way if a student arrives late they can restore the environment to the Challenge of choice.
4. Plan to update WTH delivery for One-day, Two-day and Three-day (S,M,L) sessions.  This way we tailor delivery to students' proficiency and time investment.

### CODE REVIEW

File: 019-ThisOldDataWarehouse/Student/Challenge-00.md [hunk 1]
```diff
@@ -1,34 +1,42 @@
...
+  - [Azure Cloud Shell](../../000-HowToHack/WTH-Common-Prerequisites.md#azure-cloud-shell)
+- [Visual Studio Code](../../000-HowToHack/WTH-Common-Prerequisites.md#visual-studio-code)
+  - [VS Code plugin for ARM Templates](../../000-HowToHack/WTH-Common-Prerequisites.md#visual-studio-code-plugins-for-arm-templates)
+- [Azure Storage Explorer](../../000-HowToHack/WTH-Common-Prerequisites.md#azure-storage-explorer)
 
-## Pre-requisites
-1. Laptop computer as Development environment
-2. Azure Subscription
+## Student Resources
+Your coach will provide you with a Resources.zip file that contains resource files you will use to complete some of the challenges for this hack.
```
Comment (raccchel) [inline 019-ThisOldDataWarehouse/Student/Challenge-00.md:25]: It would be better to upload a Resources.zip file in the github resources folder.

**Review by raccchel — CHANGES_REQUESTED** [review]
Make a note on the part needed to be edited

File: 019-ThisOldDataWarehouse/Student/Challenge-00.md (hunk 1 above)
Comment (jrzyshr) [reply 019-ThisOldDataWarehouse/Student/Challenge-00.md:25]: This is by design as per the WTH contributors guide.  Resolving this as-is.

File: 019-ThisOldDataWarehouse/Student/Challenge-00.md [hunk 2]
```diff
@@ -1,34 +1,42 @@
...
+
+For your local PC, ensure the following tools are installed.
 1. [SQL Server Management Studion (Version 18.x or higher)](https://docs.microsoft.com/en-us/sql/ssms/download-sql-server-management-studio-ssms?view=sql-server-ver15)
 2. [Visual Studio Code](https://code.visualstudio.com/Download) 
 3. [Power BI Desktop](https://www.microsoft.com/en-us/download/details.aspx?id=58494)
 
-
 ### Setup Azure Tenant with Services for What the Hack
 
 WWI runs their existing database platforms on-premise with SQL Server 2017.  There are two databases samples for WWI.  The first one is for their Line of Business application (OLTP) and the second is for their data warehouse (OLAP).  You will need to setup both environments as our starting point in the migration.
```
[the remark this answers is not in the document]
Comment (jrzyshr) [reply 019-ThisOldDataWarehouse/Student/Challenge-00.md]: Fixed!

**Review by jrzyshr — APPROVED** [review]
Thanks team for the massive update to one of our "well aged" hacks!
-Peter

**Pull request merged**